nutkit/protocol/responses.py

```diff
@@ -71,6 +71,36 @@ def __init__(self, id, address):
         self.address = address
 
 
+class BookmarksSupplierRequest:
+    """
+    Represents a bookmark supplier in the Bookmark Manager.
+
+    This means the Bookmark Manager is triggering the call
+    for enrich the bookmarks with user bookmarks for a given
+    database.
```

Comment:
```suggestion
    Represents a bookmark supplier in the Bookmark Manager.
    This means the Bookmark Manager is calling a user-function to get
    additional bookmarks to enrich its own set of bookmarks before using them
    for a given database.
```

Reply:
Or, if this is even the case:

```suggestion
    Represents a bookmark supplier in the Bookmark Manager.
    This means the Bookmark Manager is calling a user-function to get
    additional bookmarks to enrich its own set of bookmarks before using them
    for a given database. If database is None, the bookmark manager requests
    bookmarks for all databases.
```